osu.Game/Screens/Play/Player.cs

```diff
@@ -402,7 +411,8 @@ private void fadeOut(bool instant = false)
         {
             float fadeOutDuration = instant ? 0 : 250;
             this.FadeOut(fadeOutDuration);
-            Background?.FadeColour(Color4.White, fadeOutDuration, Easing.OutQuint);
+            Background.EnableUserDim.Value = false;
+            storyboardContainer.StoryboardReplacesBackground.Value = false;
```

Comment:
any point in this?

Reply:
When exiting player, if player had a storyboard that disables the background, whatever screen it suspends/exits to needs to have a visible background.

I added a new test after realizing this.

Reply:
This shouldn't be set on this bindable though, since this container is basically out of scope at this point (see my other comments).

Reply:
will set on the local private bindable inside Player then